packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -24,204 +24,262 @@ _ref:
+          message: |
+            <h3> YAML file extensions </h3>
+            <p> Both files with the <code>.yaml</code> and <code>.yml</code> file extensions are supported as YAML files. </p>
+            <h3> JSON instead of YAML </h3>
+            <p> Since you can reference JSON files, you can build your app using JSON instead of YAML files. The <code>lowdefy.yaml</code> file needs to be a YAML file, but all other configuration can be in referenced JSON files. It also makes sense to use JSON instead of YAML if you are generating configuration using code. </p>
 
-      - id: md2
+      - id: md3
         type: MarkdownWithCode
         properties:
           content: |
-            # Global
+            ## Config
 
-            Any data that you wish to use in your app can be stored in the __global__ object, and accessed using the [`_global`](/_global) operator. This is a good place to store data or configuration that is used throughout the app, for example the url of a logo or configuration of a page, since then these are only written once, and can be updated easily.
+            The config object has the following properties:
 
-            The global object can also be modified using the [`SetGlobal`](/SetGlobal) action.
+            - `basePath: string`: Set the base path to serve the Lowdefy application from. This will route all pages under `https://example.com/<base-path>/<page-id>` instead of the default `https://example.com/<page-id>`. The basePath value must start with "/".
+            - `homePageId: string`: The pageId of the page that should be loaded when a user loads the app without a pageId in the url route. This is the page that is loaded when you navigate to `yourdomain.com`.
 
-            # Menus
+            ## Auth
 
-            Menu objects are an object with links to pages. The are used by blocks like `PageSiderMenu` to render the links in the menu. If no menus are provided, a default menu is created, with links to all of the defined pages, and with their pageIds as menu item titles.
+            [TODO]: <> (# TODO: Add auth things here)
 
-            Blocks like like `PageSiderMenu` use the menu with `id: default` by default. This means that if you create a menu object with `id: default`, this will be used unless another menu is configured.
+            ## Global
 
-            More than one menu can be configured in an app. As an example, this can be used when two logically different sections in the app need different menus.
+            Any data that you wish to use in your app can be stored in the __global__ object, and accessed using the [`_global`](/_global) operator. This is a good place to store data or configuration that is used throughout the app, for example the url of a logo or configuration of a page, since then these are only written once, and can be updated easily.
```

Comment:
In a Lowdefy app, you can define global variables in the __global__ object, which can be accessed using the [`_global`](/_global) operator.

Reply:
Change has been made in #1314.